### ISSUE TITLE
[Bug] Kasm website don't allow to register a new account using a new gTLD.

### ISSUE THREAD

**Issue by calendulish** [kasmtech/workspaces-issues#576]
When I try to register using my email sonand@lara.monster it throw the error "Invalid email address"

**To Reproduce**
1. Go to https://app.kasmweb.com/#/signup
2. Try to register using an email that uses a new gTLD.
3. See error

**Additional context**
As you know, the new Internet extensions initiative was decided in **2009** and, as of 2013, numerous [new domain extensions](https://icannwiki.org/New_gTLD_Program) have been added to the Internet root. There’s no doubt that gTLDs have become increasingly accepted. I’ve seen how social media giants like Twitter, Facebook, and LinkedIn have come to consider them as valid. Therefore, considering **more than one decade have passed**, I strongly believe that, at this point, there is no reason why domain-specific email addresses is being blocked on kasm.

**Comment by teja-kasm** [comment]
We’re currently enhancing our GitHub issue tracker to improve organization and efficiency.

As part of this process, we’re closing older issues to help keep the tracker focused on current and active topics.

If this issue is still relevant or you’re still experiencing the problem, please feel free to create a new issue.

Thank you for your understanding!
